openslides/core/static/templates/index.html

```diff
@@ -3,7 +3,8 @@
 <meta charset="utf-8">
 <meta http-equiv="X-UA-Compatible" content="IE=edge">
 <base href="/">
-<title>OpenSlides</title>
+<title ng-if="!activeAppTitle">OpenSlides</title> <!-- avoid {{ ... }} in title while angular is not loaded -->
+<title ng-if="activeAppTitle">OpenSlides &ndash; {{ activeAppTitle | translate }}</title>
```

Comment:
This [patch.txt](https://github.com/OpenSlides/OpenSlides/files/627233/patch.txt) works, so we can mark the strings, I think.